src/handlers.cpp

```diff
@@ -1120,6 +1138,24 @@ std::string ImpuRegDataTask::server_name_from_body(std::string body)
   }
 }
 
+std::string ImpuRegDataTask::wildcard_from_body(std::string body)
+{
+  rapidjson::Document document;
+  document.Parse<0>(body.c_str());
+
+  if (!document.IsObject() ||
+      !document.HasMember("wildcard-identity") ||
```

Comment:
I've made the -/_ consistent within a request (but they are different for LIRs/SARs)